imagetest/utils/test_utils.go

```diff
@@ -129,3 +129,25 @@ func DownloadPrivateKey(user string) ([]byte, error) {
 	}
 	return privateKey, nil
 }
+
+// GetHostKeysFromDisk read ssh host public key and parse
+func GetHostKeysFromDisk() (map[string]string, error) {
+	bytes, err := ioutil.ReadFile("/etc/ssh/ssh_host_*_key.pub")
```

Comment:
`ioutil.ReadFile` doesn't support reading globs. did you run this code?

Reply:
Explicit read three key files.

Reply:
no, you have to use a glob to find what files are there, and read them all. you just can't do it with ioutil.ReadFile, because it expects a single file.